Update choose-web-site-cloud-service-vm.md

CESARDELATORRE opened the pull request:
The doc says "Automatic OS updates are planned for a future Service Fabric release.". 
OS patching of SF nodes/VMs is available through the POA service although not fully automated by Azure, which will come in next releases.
https://blogs.msdn.microsoft.com/azureservicefabric/2017/05/22/os-patching-of-service-fabric-cluster-nodes/

azurecla commented:
Hi __@chaosrealm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

CESARDELATORRE committed:
Update choose-web-site-cloud-service-vm.md

The doc says "Automatic OS updates are planned for a future Service Fabric release.". 
OS patching of SF nodes/VMs is available through the POA service although not fully automated by Azure, which will come in next releases.
https://blogs.msdn.microsoft.com/azureservicefabric/2017/05/22/os-patching-of-service-fabric-cluster-nodes/
